backup/restore/main.go

```diff
@@ -74,7 +74,7 @@ func main() {
 	backupExecutor := backup.BackupManager{
 		ETCDClient: client,
 		Provider:   backupProvider,
-		Logger:     logger,
+		Logger:     logger.SugaredLogger,
```

Comment:
why not do logging.Logger?

Reply:
Logger forces arguments to be strings. SugaredLogger can handle other types at a small overhead cost of type checking

Reply:
Because it needs a logger of type SugaredLogger